### ISSUE TITLE
Manually calculate points?

### ISSUE THREAD

**Issue by dylanavery720** [jaebradley/basketball_reference_web_scraper#67]
Hey @jaebradley ,

First of all, this is one of the coolest things ever. Really having fun with it. Thank you so much for making it!

I was just wondering if you were calculating points manually by adding free throws, field goals, and three pointers together? I'm trying to do PPG but don't see a Points value on a player when retrieving season totals. 

I don't mind doing it manually but just want to make sure I'm not missing something.

Thanks again!

Dylan

**Comment by jaebradley** [comment]
@dylanavery720 thanks for opening this issue and I'm glad you're enjoying the library!

I intentionally did not add all points together as all the relevant information is there for the API end user if they choose to aggregate that information.

An argument *could* be made that a field should be added that does the calculation, but I'm generally hesitant to add fields that could be calculated as it feels like there's duplicative (i.e unnecessary) information being contained in the same payload.

**tl;dr: No, you're not missing anything 😉**
